.github/actions/paths-filter/action.yml

```diff
@@ -43,10 +55,18 @@ runs:
         java-code-change:
           - '**/src/main/**'
           - '**/src/test/**'
-          
+
         maven-change:
           - 'bom/*'
           - 'build-tools/**'
           - 'dist/**'
           - 'parent/*'
           - 'pom.xml'
+
+        docker-change:
+          - 'Dockerfile'
+          - '*.Dockerfile'
```

Comment:
❓ This would mean if we change Operate or Zeebe docker we would also build the Camunda docker image? Is this intended?. Maybe we split the check somehow? Like General docker and subcomponents docker

Reply:
I had the same thoughts, but I am afraid we will end-up with many granular filters that we need to maintain
I will change the filter to match `camunda.Dockerfile` only

Reply:
If in doubt reusing an existing (too broad) filter is better than introducing many new ones - individual changes to Dockerfiles are rare and dependency updates will affect all of them at once, so the current filter is 👍🏼 from my POV.

Reply:
Fine for me as well, we will see whether it is an issue or not